src/main/java/mate/academy/bookstore/handler/ResponseData.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.bookstore.handler;
+
+import java.util.List;
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+public class ResponseData {
```

Comment:
```suggestion
public class ErrorResponse {
```

Reply:
No, I am going to use this in future for all responses